src/themes/rhdh/components.ts

```diff
@@ -97,6 +97,20 @@ export const components = (themeConfig: ThemeConfig): Components => {
     };
   }
 
+  // MUI AppBar
+  if (options.headers !== "mui") {
+    components.MuiAppBar = {
+      styleOverrides: {
+        root: {
+          backgroundColor:
+            general.globalHeaderBackgroundColor ??
+            general.sidebarBackgroundColor,
+          backgroundImage: "none",
```

Comment:
Can we just call it appBar here?

I know I said globalHeaderBackground ?? sidebarBackground somewhere. But there I expected that you will do that in the global-header plugin.

In the theme I think it's better to not fallback to the sidebar automatically. Instead of that we can just configure the defaults in  a way that they match.

```suggestion
          backgroundColor: general.appBarBackgroundColor,
          backgroundImage: general.appBarBackgroundImage,
```

Or should we use?

```suggestion
          backgroundColor: general.appBar?.backgroundColor,
          backgroundImage: general.appBar?.backgroundImage,
```

I know we don't use this for the others yet. Hmmm